_toc.yml

```diff
@@ -0,0 +1,58 @@
+format: jb-book
+root: README
+
+parts:
+
+  - caption: About These Tutorials
+    chapters:
+    - file: doc/HowToUse                                                           # TODO
+    - file: environment_set_up/README                                              # TODO: this doc is in-process on another branch
+
+
+  - caption: Common Workflows
+    chapters:
+    - file: dataset_preprocessing/README                                           # TODO
+      sections:
+      - file: dataset_preprocessing/ReChunkingData
+      - file: dataset_preprocessing/ReChunkingData_Cloud
+      # TODO: Add more pre-processing sections
+    # TODO: Add more  common/universal chapters
+
+
+  - caption: Forcings Workflows                                                    # TODO:  This whole PART needs fleshing out...
+    chapters:
+    - file: dataset_access/conus404_explore   # placeholder only
+
+
+  - caption: Evaluation Workflows
+    chapters:
+    - file: model_evaluation/tutorials/nwm_streeamflow/00_Overview
+      sections:
```

Comment:
I think I'd like to see a section per evaluation workflow. So maybe we need an explainer per eval workflow (like you did with the 00_Overview file). This could be a markdown file that says "This workflow compares NWM v2.1 to NWIS data at ___ gage locations" or something like that. Or we could just have a section title that is descriptive enough to show what is being compared in the workflow and it just points to a top level directory containing these files. But there needs to be a way in the TOC to group these sets of notebooks together. @alaws-USGS will be developing equivalent workflows for different datasets that will need to sit beside this in the TOC.

Reply:
Maybe Model Evaluation is actually a "part" and NWM eval is a chapter, then the 4 notebooks are sections.